packages/cli/src/lib/Web3API.ts

```diff
@@ -1,8 +1,9 @@
-import { Manifest } from "./Manifest";
-
 import YAML from "js-yaml";
 import fs from "fs";
 
+import { Manifest } from "./Manifest";
+import { manifestValidation } from "./helpers/validators";
```

Comment:
I think manifest validation should be inside of the client package, and the CLI can import it from there. Clients will need to validate manifests at run-time when they download new Web3APIs.

Reply:
Ok ok makes sense